workbench/config/WorkbenchConfig.php

```diff
@@ -60,17 +60,22 @@ function __construct() {
             $envKey = str_replace("___DOT___", ".", $envKey);
         
             $envKeyParts = explode($configDelim, $envKey);
-        
+
+            $lastKey = end(array_values($envKeyParts));
```

Comment:
@rvwatch - Actually let's **not** do this. The old way, while a bit clunky seems to be fastest. According to https://theghostwhocodes.com/php-how-to-get-the-last-element-of-an-array-2a0db5343520

> end(array_values($array)); It’ s the worst solution, both in terms of elegance and performance: it takes 176.85 s! Fast using end to work on array internal pointer, but terribly slow creating a new array every time with array_values, used only to avoid the reset call.

Reply:
Ok. I'll switch it back up.